src/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Client Diversity
+description: A high level explanation of the importance of Ethereum client diversity.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+## Introduction {#Introduction}
+
+The behaviour of an Ethereum node is controlled by the client software it runs. There are several production-level Ethereum clients, each one developed and maintained in different languages by separate teams. The clients are built to a common spec that ensures the clients seamlessly communicate with each other and have the same functionality and provide an equivalent user experience. Multiple clients exist because the development community has decided to build multiple implementations as a way to strengthen the network. However, at the moment the distribution of clients across nodes is not even enough to realise this network fortification to its full potential. Ideally, users divide roughly equally across the various clients to bring as much client diversity as possible to the network.
+
+## Prerequisites {#prerequisites}
+
+If you don't already have an understanding of what nodes and clients are, check out [Nodes and clients](/developers/docs/nodes-and-clients/). The Beacon Chain is explained [here](/src/content/eth2/beacon-chain/index.md). Execution and consensus layers are defined in the [glossary](/src/content/glossary/index.md).
+
+## Current Client Diversity? {#current-client-diversity}
+
+![Diagram showing snapshot of client diversity](../client-diversity.jpg)
+_This diagram shows snapshots of client diversity replotted using data from Ethernodes.org (execution layer) and from Michael Sproul (https://github.com/sigp/blockprint) for the consensus layer_
+
+The two pie charts above show snapshots of the current client diversity for the execution and consensus layers (at time of writing in January 2022). The execution layer is overwhelmingly dominated by [Geth](https://geth.ethereum.org/), with [Open Ethereum](https://openethereum.github.io/) a distant second, [Erigon](https://github.com/ledgerwatch/erigon) third and [Nethermind](https://nethermind.io/) fourth, with other clients comprising less than 1 % of the network. The most commonly used client on the consensus layer - [Prysm](https://prysmaticlabs.com/#projects) - is not as dominant as Geth but still represents over 60% of the network. [Lighthouse](https://lighthouse.sigmaprime.io/) and [Teku](https://consensys.net/knowledge-base/ethereum-2/teku/) make up ~20% and ~14% respectively, and other clients are rarely used.
```

Comment:
other option is embedding some live data from clientdiversity.org's api (for consensus layer only atm), but I'll delegate this decision up to the web team who might actually be able to implement this!

Reply:
Yeah, I guess I have some hesitation in sticking a "Current client diversity" section in the middle of this explainer. 

Client diversity as a topic is important in general... feel like we should focus on the broader topic at first, then get into this further down. Perhaps move this section just above the "Use a minority client" section below? This could help organize the page into the info that changes less or more frequently, so it's easier to maintain in the future.

Reply:
I agree with this. It makes the page more maintainable and makes more sense to me from an information architecture point of view.

1. **High-level understanding:** Why are there multiple clients?
2. **Potential problems:** Why is client diversity important?
3. **The state of play:** Current client diversity
4. **This solution:** Use a minority client 

In the current structure, points 2 and 3 are reversed which means people are reading about the current state of play without really understanding why this might be bad.